modules/javafx.graphics/src/main/native-glass/win/GlassClipboard.cpp

```diff
@@ -29,6 +29,12 @@
 #include <hash_map>
 #include <hash_set>
 
+// The following is derived from _HASH_SEED, which is an internal constant from
+// include/xhash; since this is an internal constant we define our own rather
+// that relying on something that Microsoft could remove (and apparately has in
```

Comment:
nitpick: typo: apparately -> apparently

Reply:
Good on you, found one I missed! 👍

Reply:
fixed that as well as one other on this line:

that --> than

Reply:
That's what I get for committing it quickly without proofreading it first. Three typos in one comment block. :)